modules/ROOT/pages/sdk-xattr-overview.adoc

```diff
@@ -1,15 +1,26 @@
 = Extended Attributes
 
 [abstract]
-Couchbase Server permits the definition of _extended attributes_.
-These allow application developers to define application-specific metadata that will only be visible to those applications that request it or attempt to modify it.
-This might be, for example, meta-data specific to a programming framework that should be hidden by default from other frameworks, or possibly from other versions of the same framework.
+Extended Attributes (XATTR) are metadata that can be provided on a per-application basis.
+
+include::6.0@sdk:shared:partial$sdk-xattr-overview.adoc[tag=intro_extended_attributes]
 
 include::6.0@sdk:shared:partial$sdk-xattr-overview.adoc[tag=using_extended_attributes]
 
-== Language-Specific Variations
 
-Each language supported by the Couchbase SDK implements extended-attribute support in a slightly different way.
-For Java examples, see xref:sdk-xattr-example.adoc[Sample Code].
+== Virtual Extended Attributes
+
+_Virtual_ extended attributes consist of meta-data on an individual document: this can be retrieved by specifying `$document` as a search-path.
+For example, `bucket.lookupIn(key).get("$document", new SubdocOptionsBuilder().xattr(true)).execute()`. 
+See xref:6.0@server:learn:data/extended-attributes-fundamentals.adoc#virtual-extended-attributes[the Virtual XATTR Section] for more information on the meta-data that they expose.
+
+These attributes are generated on-demand to expose storage-level document metadata, such as TTL to expose document expiration.
+For TTL using Virtual XATTR, use the following:
```

Comment:
This feels like a sentence fragment?

Reply:
It's within our current style guide, but happy to discuss alternatives.